stripe/src/main/java/com/stripe/android/view/CardNumberEditText.kt

```diff
@@ -177,6 +177,8 @@ class CardNumberEditText internal constructor(
         }
 
         updateLengthFilter()
+
+        this.layoutDirection = LAYOUT_DIRECTION_LTR
```

Comment:
Should this be done in `StripeEditText`?

Reply:
That feels like a wide sweep, like if you used it for email address, that wouldn't work.

Reply:
That makes sense. Should we do this for `PostalCodeEditText`?

Reply:
Also can we add a comment in the card widgets that they are LTR only if we don't already have that?

Reply:
I did a little research, and from what I found the postal code edit text is functionality as other apps.  I did make a note in the comment as well that this was explicit.